compiler/testData/codegen/bytecodeText/kt10259_3.kt

```diff
@@ -37,10 +37,10 @@ inline fun test(crossinline s: () -> Unit) {
 // this behavior is equivalent to javac and seems to be correct.
 
 // JVM_IR_TEMPLATES
-// 3 INNERCLASS Kt10259_3Kt\$box\$1\$1\s
+// 2 INNERCLASS Kt10259_3Kt\$box\$1\$1\s
```

Comment:
Templates in JVM_IR_TEMPLATES and JVM_TEMPLATES mostly looks identical. Could you extract common part in header, e.g.
// COMMON_TEMPLATE_1
...

// JVM_TEMPLATES
// JVM_PATTERN_1
...
//JVM_IR_TEMPLATES
// JVM_IR_TEMPLATE_1
...

here and in other affected files

Reply:
Didn't know that we can put common part ahead. Done in that way, but not that many due to the slightly different inner-class names.

Reply:
Oh..I expected more commons, sorry